README.md

```diff
@@ -43,8 +43,9 @@ Breaking Changes!
 
 ## Usage
 
-* One can visit `/_ping.php` to get a status
-* By using `?debug=hash` query, one can get status check status table, and time profiling table. The `hash` is 4 first letters of the salt in `settings.php`.
+* Visit `/_ping.php` to get a system status
+* By using `?debug=hash` additional status check table and time profiling information is displayed.
```

Comment:
Actually, we should replace `hash` with `token`.
Because now it's way more sophisticated than just hash.
The token can be acquired by `php _ping.php`.